scripts/balances.py

```diff
@@ -0,0 +1,63 @@
+import dbs
+import bson
+
+golos_accounts = dbs.golos_db['account_object']
+
+def save_balances(doc):
+    cyberway_accounts = dbs.cyberway_eosio_token_db['accounts']
+    cyberway_balances = dbs.cyberway_gls_vesting_db['balances']
+
+    if (cyberway_accounts.find({"_SCOPE_":doc["name"]}).count() or
+            cyberway_balances.find({"_SCOPE_":doc["name"]}).count()):
+        return
+
+    balance = {
+	"balance": {
+	    "amount": doc["balance_value"],
```

Comment:
Amount in Golos database - it is float.When converting, it is necessary to convert it into int based on decs.
decs is 3 for token (`balance_value`) and 6 for vesting (values in gls.vesting table).
I propose to define constants for names and precision, which then use.
In Cyberway token and vesting has identically names, so: GOLOS -> GLS, GESTS -> GLS (for testnet)